.github/workflows/pre-release.yml

```diff
@@ -51,14 +51,12 @@ jobs:
         run: |
           yarn install
 
-      - name: Run full end to end tests
+      - name: Run full production end to end tests
         run: |
-          cp src/test/resources/config/keystore.p12 src/main/resources/config/keystore.p12
+          docker-compose -f src/main/docker/postgresql.yml up -d
           ./gradlew bootRun &>mp.log </dev/null &
-          yarn run wait-for-managementportal
-          ./gradlew generateOpenApiSpec
-          yarn e2e
-          ./gradlew --stop
+          yarn run e2e-prod
```

Comment:
do you need to wait here for the services to startup?

Reply:
the yarn e2e command has a built in wait period :)